msal_extensions/libsecret.py

```diff
@@ -13,33 +13,29 @@
     pip install wheel
     PYGOBJECT_WITHOUT_PYCAIRO=1 pip install --no-build-isolation pygobject
 """
-import logging
-
-logger = logging.getLogger(__name__)
 
 try:
     import gi  # https://github.com/AzureAD/microsoft-authentication-extensions-for-python/wiki/Encryption-on-Linux  # pylint: disable=line-too-long
 except ImportError:
-    logger.exception(
-        """Runtime dependency of PyGObject is missing.
+    raise ImportError("""cannot import name 'gi' from 'libsecret' ({})
+Runtime dependency of PyGObject is missing.
 Depends on your Linux distro, you could install it system-wide by something like:
     sudo apt install python3-gi python3-gi-cairo gir1.2-secret-1
 If necessary, please refer to PyGObject's doc:
 https://pygobject.readthedocs.io/en/latest/getting_started.html
-""")
-    raise
+""".format(__file__))  # Message via exception rather than log
```

Comment:
`__file__` will already be in the traceback

Reply:
I was merely trying to mimic the behavior from the underlying `gi` library. It happens to include the file name in the ImportError message, even when it is also available from the traceback.

`ImportError: cannot import name '_gi' from 'gi' (/home/demo/Live-usb-storage/env37/lib/python3.7/site-packages/gi/__init__.py)`

That being said, I agree with you that we don't have to do that. I'll remove the `__file__` thing in my next commit.